backend/corpora/dataset_processing/process.py

```diff
@@ -397,16 +406,18 @@ def copy_cxg_files_to_cxg_bucket(cxg_dir, object_key, cellxgene_bucket):
 
 
 def convert_file_ignore_exceptions(
-    converter: typing.Callable, local_filename: str, error_message: str
+    converter: typing.Callable, local_filename: str, error_message: str, dataset_id: str, processing_status_type: str
 ) -> typing.Tuple[str, ConversionStatus]:
+    logger.info(f"Converting {converter}")
```

Comment:
Should we add a corresponding `Finished converting xyz in x ms` log statement so that we can start collecting some performance stats?